Assets/Scripts/AchievementStatusIconScript.cs

```diff
@@ -69,4 +69,8 @@ void Update()
             }
         }
     }
+
+	public void OnDeselect(BaseEventData eventData){
```

Comment:
Interesting putting the sfx on deselect rather than select, smart!

Reply:
Hacky but it works haha